client/gutenberg/editor/main.jsx

```diff
@@ -21,15 +22,19 @@ import { translate } from 'i18n-calypso';
 import './hooks'; // Needed for integrating Calypso's media library (and other hooks)
 import isRtlSelector from 'state/selectors/is-rtl';
 import refreshRegistrations from '../extensions/presets/jetpack/utils/refresh-registrations';
+import { getSiteOption } from 'state/sites/selectors';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
+const setDateGMTOffset = offset =>
+	setDateSettings( set( __experimentalGetSettings(), [ 'timezone', 'offset' ], offset ) );
```

Comment:
should we create a test for this so we get some notice before it disappears?

```js
describe( 'core imports', () => {
	test( 'experimental get settings exist', async () => {
		const wpImport = await import( '@wordpress/date' );

		expect( wpImport ).toHaveProperty( '__experimentalGetSettings', expect.any( Function ) );
	} );
} );
```

or something like that